### DISCUSSION TITLE
TS1036: Statements are not allowed in ambient contexts

### DISCUSSION THREAD

**Discussion by ganeshan102** [sensedeep/dynamodb-onetable#143] [Q&A] [answered]
Hi,

Currently we are facing the below issue with Typescript compiler.

node_modules/dynamodb-onetable/dist/mjs/Model.d.ts:56:2 - error TS1036: Statements are not allowed in ambient contexts.
56 };

**Comment by mobsense** [comment, marked as answer]
That should be fixed in 1.9
